araproc/analysis/hilbert_envelope_snr.py

```diff
@@ -0,0 +1,110 @@
+# this calculates Hilbert envelope SNR for each channel and their average
+
+import numpy as np
+from scipy.signal import hilbert
+
+from araproc.analysis import snr
+from araproc.framework import waveform_utilities as wfu
+
+
+
+def get_envelope(waveform, nsegs = 8):
```

Comment:
I think we can drop `nsegs` from the arguments.

Reply:
I kept 'nsegs' in case we want to use the commented portion script. But yeah, I can move that inside the commented part.

Reply:
Ah I see now. I think the commented part isn't going to be used at this point, so probably better to remove the comment block too to make it less cluttered.